### PR TITLE
French fix

### PR DESCRIPTION
**Pull request by ludoo0d0a**
download script from crowdin
+ last fixes on french

remains urls for manual issue

### CODE REVIEW

File: script/get-langs.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# set -x
+TOKEN=dfda442f9e638950fff0b7dadfa57b076b559a408bc29ceca7d014476d24d4cfefa69b88a734dbd5
```
Comment (S-Man42) [inline script/get-langs.sh:2]: Are you sure, that our private token should be applied to the public repo?

Comment (ludoo0d0a) [reply script/get-langs.sh:2]: arghh...

Comment (ludoo0d0a) [reply script/get-langs.sh]: I revoked it

**Comment by S-Man42** [comment]
Well the problem now is, it is shown in the history and already in the discussion here. Is it possible to generate a new token?

**Comment by ludoo0d0a** [comment]
> Well the problem now is, it is shown in the history and already in the discussion here. Is it possible to generate a new token?

already done => revoked + generate a new one in .gitignored file.

remove commit is painful ; i choose easiest solution

**Comment by S-Man42** [comment]
Unfortunately this branch cannot be merge due to some conflicts in en.json. Please resolve them.

**Comment by S-Man42** [comment]
There's a new sub directory: "external_helpers". You can put your script there under "external_helpers/crowdin/" or something

**Comment by S-Man42** [comment]
We have still a conflict on en.json :(

**Pull request merged**